lib/core/jbackend.c

```diff
@@ -68,32 +69,31 @@ j_backend_load (gchar const* name, JBackendComponent component, JBackendType typ
 	g_free(tpath);
 	g_free(path);
 #endif
-
```

Comment:
Why all the whitespace changes?

Reply:
Whoops i messed up my cherry pick and squash. Force pushed a fixed commit.